### PR TITLE
feat: move already have account message above content

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR moves the "Already have an account? Sign in" message next to the Title. It makes it more visible for existing users where to sign in.

__After__

after

__Before__

before

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test in the editor
3. Check front-end

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/blocks/reader-registration/style.scss [hunk 1]
```diff
@@ -82,19 +95,35 @@
...
+		background: var( --newspack-secondary-color );
+		color: var( --newspack-secondary-contrast-color );
+		display: flex;
+		font-size: 1em;
+		justify-content: center;
+		left: 0;
+		line-height: 1.5;
+		min-height: 36px;
+		padding: 0 16px;
+		position: absolute;
```
Comment (adekbadek) [inline assets/blocks/reader-registration/style.scss:108]: If the Registration block is placed on a page (not in a prompt), this makes the banner stick to the top of the viewport.

File: assets/blocks/reader-registration/style.scss [hunk 2]
```diff
@@ -190,6 +225,27 @@
 	}
 }
 
+/* Sepcific styles to apply to .newspack-popup when Reader Registration is displayed */
```
Comment (adekbadek) [inline assets/blocks/reader-registration/style.scss:228]: There's no conditional logic here – these styles will be applied to prompts regardless of their content:

image

**Review by adekbadek — CHANGES_REQUESTED** [review]
I don't think it's a good idea to make this banner absolutely positioned. With the background color and placed above the email input, it's prominent enough.

File: assets/blocks/reader-registration/style.scss (hunk 1 above)
Comment (thomasguillot) [reply assets/blocks/reader-registration/style.scss]: I completely reworked this in https://github.com/Automattic/newspack-plugin/pull/1967/commits/207127595e4426fea51c90cdc76c8bc5f0951fa7

- I didn't like having a "floating" banner at the top. Instead, I moved the messaging next to the block title. So it's similar to the Sign In modal.

File: assets/blocks/reader-registration/edit.js [hunk 3]
```diff
@@ -206,15 +206,35 @@ export default function ReaderRegistrationEdit( {
...
-							<RichText
-								onChange={ value => setAttributes( { title: value } ) }
-								placeholder={ __( 'Block title…', 'newspack' ) }
-								value={ title }
-								tagName="h2"
-							/>
+							<div className="newspack-registration__header">
+								<RichText
+									onChange={ value => setAttributes( { title: value } ) }
+									placeholder={ __( 'Write title…', 'newspack' ) }
```
Comment (adekbadek) [inline assets/blocks/reader-registration/edit.js:212]: Nit: the tone here feels weird, how about 

```suggestion
									placeholder={ __( 'Add title…', 'newspack' ) }
```

like the WP post editor? And then same with the description input.

**Review by adekbadek — APPROVED** [review]
Added one wording suggestion.

File: assets/blocks/reader-registration/edit.js (hunk 3 above)
Comment (thomasguillot) [reply assets/blocks/reader-registration/edit.js]: I copied the wording from the Cover block but I guess there are a lot of inconsistencies between blocks....
"Write content...", "Heading", "Add quote", "Content..."

Changed to "Add title" and "Add description" (no elipsis) in https://github.com/Automattic/newspack-plugin/pull/1967/commits/12c1a139669fd392e57d697f92c62cbab3096bfe

**Pull request merged**